bin/review-rot

```diff
@@ -0,0 +1,130 @@
+                request pull requests
+                """
+                res = format_user_repo_name(data)
+                """
+                get pull requests for specified git service, username
+                and repository name
+                """
+                if not item.has_key('token'):
+                    log.debug('token not found for %s', item['type'])
+                    raise KeyError("token not found for %s. Please provide 'None' "
+                                   "if not applicable"  % item['type'])
+                if not item.has_key('host'):
+                    log.debug('host not found for %s', item['type'])
+                    raise KeyError("host not found for %s. Please provide 'None' "
+                                   "if not applicable"  % item['type'])
+                git_service.request_reviews(user_name=res['user_name'],
+                                            repo_name=res['repo_name'],
+                                            state_=state_,
+                                            value=value,
+                                            duration=duration,
+                                            token=item['token'],
+                                            host=item['host']
+                                            )
+
+
+def format_user_repo_name(data):
+    """
+    Takes input from configuration file for a specified git service.
+    Split or fomat it as per required.
+    Args:
+        data (str): combination of username and/or reponame
+
+    Returns:
+        Dictionary representation of username and reponame
+    """
+    if '/' in data:
+        # Splitting only once in case "/" is a value character in the data.
+        user_name, repo_name = data.split('/', 1)
+
+    else:
+        user_name = data
+        repo_name = None
+    return {'user_name': user_name, 'repo_name': repo_name}
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Lists pull/merge requests for github, gitlab,'
+                    'pagure and gerrit')
+    parser.add_argument('-s', '--state',
+                        default=None,
+                        choices=['older', 'newer'],
+                        help="Pull requests state 'older' or 'newer'"
+                        )
+    parser.add_argument('-v', '--value',
+                        default=None,
+                        type=int,
+                        help='Pull requests duration in terms of value(int)'
+                        )
+    parser.add_argument('-d', '--duration',
+                        default=None,
+                        choices=['y', 'm', 'd', 'h', 'min'],
+                        help='Pull requests duration in terms of y=years,'
+                             'm=months, d=days, h=hours, min=minutes')
+    parser.add_argument('--debug', action='store_true',
+                        help='Display debug logs on console')
+    args = parser.parse_args()
+    options = []
```

Comment:
Another, perhaps more robust, way of doing this might be:

```python
options = (args.state, args.value, args.duration)
if any(options) and not all(options):
    parser.error('Either none or all arguments are required')
```